### PR TITLE
Ready for Review

### PR DESCRIPTION
**Pull request by maelfosso**
Hello,

First, sorry for the message **Can't automatically merge**. I started working on the project alone by waiting for a new partner and when they notify me that there is not yet a partner I typed a few Git commands to arrange everything according to Microverse Guidelines. 
After the review, I will merge and solve the conflicts.

Please, could review my project ?
Thanks.

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+function changeReadStatusFromLibrary(idx) {
+  let book = myLibrary[idx];
+  book.toggleRead();
+  myLibrary.splice(idx, 1, book);
+  //
+  // let table = document.getElementById('books-list');
+  // table.deleteRow(+idx + 2);
+
```
Comment (JamesOkunlade) [inline js/app.js]: Remove commented code.

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    }
+  }
+
+  let book = new Book(
+    payload.title,
+    payload.author,
+    payload.pages
+  );
+
+  // document.getElementById("myForm").reset();
```
Comment (JamesOkunlade) [inline js/app.js]: Remove commented code.

File: js/app.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+    removeBookToLibray(bookIdx);
+  });
+}
+
+// document.getElementById('add-book').addEventListener('click', function(e) {
+//   e.preventDefault();
+//
+//
+// });
```
Comment (JamesOkunlade) [inline js/app.js]: Remove commented code

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Nice job with the library project 👏 😄 , However, you'll have to set the project up again to avoid the merge conflict and make you stickler work.

When submitting for another review, ensure to add a descriptive README.md file to your project and remove all commented code.

**Status: `Changes required`**

Happy coding! 😎

**Review by AdnanAfsari — APPROVED** [review]
Hello @maelfosso ✋ 

Nice, I think you have made all the changes the previous reviewer requested so its 👍 from me.

**Status:Approved**

Happy Coding

**Pull request merged**